hal/lp_ticker_api.h

```diff
@@ -33,13 +33,13 @@ extern "C" {
  * \defgroup hal_lp_ticker Low Power Ticker
  * Low level interface to the low power ticker of a target
  *
- * # Defined behavior
- * * Has a reported frequency between 8KHz and 64KHz - verified by lp_ticker_info_test
- * * Has a counter that is at least 12 bits wide - verified by lp_ticker_info_test
- * * Continues operating in deep sleep mode - verified by lp_ticker_sleep_test
- * * All behavior defined by the @ref hal_ticker_shared "ticker specification"
+ * # Defined behaviour
```

Comment:
The American spelling of words is used in our docs. This should be ```behavior```. CC @AnotherButler

Reply:
@c1728p9 You are correct. Our documentation uses U.S. English, which means we don't include a "u" in "behavior".

Reply:
Fixed as suggested.